fibsem/milling/core.py

```diff
@@ -281,5 +269,29 @@ def acquire_images_after_milling(
 
     return images
 
-# QUERY: should List[FibsemMillingStage] be a class? that has it's own settings? 
+
+def get_stage_reference_image(
+    microscope: FibsemMicroscope, milling_stage: FibsemMillingStage
+) -> FibsemImage:
+    ref_image = milling_stage.reference_image
+    if isinstance(ref_image, FibsemImage):
+        return ref_image
+    elif ref_image is None:
+        path = milling_stage.imaging.path
+        if path is None:
+            path = Path(fcfg.DATA_CC_PATH)
+        image_settings = ImageSettings(
+            hfw=milling_stage.milling.hfw,
+            dwell_time=1e-6,
+            resolution=[1536, 1024],
+            beam_type=milling_stage.milling.milling_channel,
+            reduced_area=milling_stage.alignment.rect,
+            path=path,
+            filename=f"ref_{milling_stage.name}_initial_alignment_{current_timestamp_v2()}",
+        )
+        return microscope.acquire_image(image_settings)
```

Comment:
```suggestion
        image_settings = ImageSettings(
            hfw=milling_stage.milling.hfw,
            dwell_time=1e-6,
            resolution=[1536, 1024],
            beam_type=milling_stage.milling.milling_channel,
            reduced_area=milling_stage.alignment.rect,
            save=True,
            path=path,
            filename=f"ref_{milling_stage.name}_initial_alignment_{current_timestamp_v2()}",
        )
        return acquire.acquire_image(microscope, image_settings)
```
In testing this, I noticed that this doesn't actually save the reference image (previously existing bug). Needs to have save=True, and use the acquire api